pkg/dashboard/assets/css/main.css

```diff
@@ -529,3 +529,79 @@ UTILITIES
   white-space: nowrap;
   width: 1px;
 }
+
+.email-box {
+  background: #4c415f;
+  padding: 1.375rem 0.8125rem;
+  width: 100%;
+}
+
+.email-box__toggle-title {
+  color: #fff;
+  font-size: var(--step-4);
+  font-weight: 600;
+}
+
+.email-box__content p {
+  color: #fff;
+  padding: 0.75rem 0 1.5rem 0;
+}
+
+.email-box__email-label {
+  position: relative;
+}
+
+.email-box__email-label span {
+  color: #d3d3d3;
```

Comment:
Based on my understanding of the Web Content Accessibility Guidelines and checking with the [WebAIM: Contrast Checker](https://webaim.org/resources/contrastchecker/), I think the contrast of this gray color against the white background is too low, making this label hard to see. Could we please increase the contrast here?

Reply:
Hi Cecilia, I updated the code. Thank you for your feedback!